docs/readmes/proposals/p0010_Ubuntu_upgrade.md

```diff
@@ -0,0 +1,89 @@
+---
+id: p0010_Ubuntu_upgrade
+title: AGW Ubuntu upgrade
+hide_title: true
+---
+
+# Overview
+
+*Status: In-Review*\
+*Author: @pshelar*\
+*Last Updated: 03/15*\
+*Targeted Release: 1.5*
+
+This document cover AGW ubuntu upgrade path
+
+## Goal
+Add support for Ubuntu based AGW.
+
+## Introduction
+
+There are multiple reasons to update base OS from current Debian to ubuntu distribution
+
+1. Python 3.5 is EOL on September 13th, 2020. Current Debian does not support the latest python release. pip 21.0
+   dropped support for Python 3.5 in January 2021.
+2. Newer kernel is required for enabling next generation datapath performance.
+3. Orc8r is already on Ubuntu. This would converge the magma platform on ubuntu distribution
+4. With newer distribution we can relax AGW kernel dependency using *DKMS packages for OVS kernel module*
+
+## Deployment scenarios for 1.5
+
+### *Fresh install on Ubuntu:*
+
+This is going to be default deployment OS for AGW from 1.5 onwards. Using ubuntu base OS for AGW 1.5 would also result
+in smoother AGW updates for future releases.
+
+### *Debian based AGW 1.4 to Debian based AGW 1.5:*
+
+Current production deployment can use current Debian upgrade path for upgrading magma from 1.4 to 1.5. Existing workflow
+remains same.
+
+### *Debian based AGW 1.4 to Ubuntu based AGW 1.5:*
+
+This deployment involves two steps first, upgrading underlying OS, second step installing AGW packages.
```

Comment:
Would this be an upgrade of the OS or would it be a reinstall?

Reply:
This is upgrade of OS.